backend/conspects/serializers.py

```diff
@@ -76,3 +76,14 @@ class Meta:
         fields = ["id", "name", "extension", "content", "can_be_edited", "can_be_previewed", "is_attachment"]
         read_only_fields = ["can_be_edited", "can_be_previewed", "is_attachment"]
 
+    def validate_edition(self, value):
```

Comment:
Why do we have ```validate_edition``` check in ```FileSerializer```? We do not have edition field in ```File``` model. Maybe it should be moved to ```FolderSerializer```